### ISSUE TITLE
Output as JSON?

### ISSUE THREAD

**Issue by ChrisJefferson** [ciaranm/glasgow-subgraph-solver#16]
It seems the output format is fixed to a simple text based thing. Would it be possible to have a flag to output JSON?

I suggest JSON not because it is great, but because every language in the world has a JSON parser, so it makes the output easier to read in (particularly the mappings)

**Comment by ciaranm** [comment]
Yeah, I've not got any massive moral objection to this. Will put it on the list of things to either do or allocate to a bored student.
